cmd/mt-split-metrics-by-ttl/main.go

```diff
@@ -32,7 +32,8 @@ var (
 	cassandraUsername = flag.String("cassandra-username", "cassandra", "username for authentication")
 	cassandraPassword = flag.String("cassandra-password", "cassandra", "password for authentication")
 
-	windowFactor = flag.Int("window-factor", 20, "the window factor be used when creating the metric table schema")
+	windowFactor             = flag.Int("window-factor", 20, "the window factor be used when creating the metric table schema")
+	cassandraOmitReadTimeout = flag.Int("cassandra-omit-read-timeout", 60, "if a read is older than this, it will directly be omitted without executing")
```

Comment:
this tool doesn't do any reads, so having this option here is rather confusing. we can just hardcode a number and comment that it's not relevant because we don't do reads.